ext/ftp/php_ftp.c

```diff
@@ -141,7 +141,7 @@ ZEND_BEGIN_ARG_INFO_EX(arginfo_ftp_nb_fget, 0, 0, 4)
 	ZEND_ARG_INFO(0, resumepos)
 ZEND_END_ARG_INFO()
 
-ZEND_BEGIN_ARG_INFO(arginfo_ftp_pasv, 0)
+ZEND_BEGIN_ARG_INFO_EX(arginfo_ftp_pasv, 0, 0, 2)
```

Comment:
Use this form because I was getting some conflicts with `master` :disappointed:

Reply:
IMHO it's okay. Most important is to fix the erroneous "1 required argument".